test/test_qa_utils.py

```diff
@@ -3,11 +3,19 @@
 import pytest
 import pytz
 import re
+import subprocess
 import time
-import unittest
 import uuid
 
 from dcicutils.qa_utils import mock_not_called, local_attrs, override_environ, ControlledTime
+from unittest import mock
+
+# The following line needs to be separate from other imports. It is PART OF A TEST.
+from dcicutils.qa_utils import notice_pytest_fixtures   # Use care if editing this line. It is PART OF A TEST.
+from .fixtures.sample_fixtures import MockMathError, MockMath, math_enabled
+
+
+notice_pytest_fixtures(math_enabled)   # Use care if editing this line. It is PART OF A TEST.
```

Comment:
So is this what you would do at the top of a test file if you want the "unused" errors to go away?

Reply:
Yep. I think it effectively says that in the doc string. Let me know if you think more is needed. Really you can do it for things that don't have unused things, too. It would be nice to know what fixtures are in use. But I think almost all fixtures will in fact look globally unused, so you'd be well-motivated. It's harder to tell within a function because some things have fixture effects but they also get used.

Reply:
Makes sense to me.